### PR TITLE
optimize 1192: Set a friendly prompt respectively when the login user does not exist and the password is wrong 

### LINKED ISSUE #1192: There is no friendly prompt when the user does not exist.
### Which Project did you use? Hippo4j config or Hippo4j server?
Hippo4j server
### Which version did you use?
1.5.0
### Expected behavior
hint "User does not exist"
### Actual behavior
hint "Error"
### Cause analysis (if possible)
image

### PR DESCRIPTION
**Pull request by lucca-suen**
Fixes #1192

Changes proposed in this pull request:
- 

> Check mailbox configuration when submitting. [Contributor Guide](https://hippo4j.cn/community/contributor-guide)

### CODE REVIEW

File: hippo4j-server/hippo4j-auth/src/main/java/cn/hippo4j/auth/toolkit/ReturnT.java [hunk 1]
```diff
@@ -43,13 +43,13 @@
 
     private int code;
 
-    private String msg;
+    private String message;
```
Comment (magestacks) [inline hippo4j-server/hippo4j-auth/src/main/java/cn/hippo4j/auth/toolkit/ReturnT.java:46]: Will the transformation here have any impact on other processes?

File: hippo4j-server/hippo4j-auth/src/main/java/cn/hippo4j/auth/filter/JWTAuthenticationFilter.java [hunk 2]
```diff
@@ -108,6 +113,19 @@ protected void successfulAuthentication(HttpServletRequest request,
...
     protected void unsuccessfulAuthentication(HttpServletRequest request, HttpServletResponse response, AuthenticationException failed) throws IOException {
         response.setCharacterEncoding("UTF-8");
-        response.getWriter().write(JSONUtil.toJSONString(new ReturnT(ReturnT.JWT_FAIL_CODE, "Server Error")));
+        response.getWriter().write(JSONUtil.toJSONString(new ReturnT(ReturnT.JWT_FAIL_CODE, getMessage(failed))));
+    }
+
+    /**
+     * Return different echo information to the front end according to different exception types
+     */
+    protected String getMessage(AuthenticationException failed) {
```
Comment (magestacks) [inline hippo4j-server/hippo4j-auth/src/main/java/cn/hippo4j/auth/filter/JWTAuthenticationFilter.java]: Is it possible to define private at method level? Also please create a unit test for it.

File: hippo4j-server/hippo4j-auth/src/main/java/cn/hippo4j/auth/toolkit/ReturnT.java (hunk 1 above)
Comment (lucca-suen) [reply hippo4j-server/hippo4j-auth/src/main/java/cn/hippo4j/auth/toolkit/ReturnT.java:46]: I noticed that in the front-end interceptor, when parsing the back-end data packet, the name of the checked field is "message", so change "msg" to "message".

If you use the old name "msg", without changing the front-end code, the front-end will not be able to obtain the information that the back-end wants to submit to the front-end.

File: hippo4j-server/hippo4j-auth/src/main/java/cn/hippo4j/auth/filter/JWTAuthenticationFilter.java (hunk 2 above)
Comment (lucca-suen) [reply hippo4j-server/hippo4j-auth/src/main/java/cn/hippo4j/auth/filter/JWTAuthenticationFilter.java]: no problem

**Review by magestacks — APPROVED** [review]

**Pull request merged**